src/containers/main-page/preview/PersonalDetails.tsx

```diff
@@ -0,0 +1,33 @@
+import { Grid } from '@mui/material';
+import PersonalInformation from './PersonalInformation';
+import Languages from './Languages';
+import Skills from './Skills';
+import Certifications from './Certifications';
+
+const PersonalDetails = function (): JSX.Element {
+  return (
+    <Grid
+      container
+      sx={{
+        maxWidth: '100%',
+        margin: 'auto',
+      }}
+    >
```

Comment:
You can use 
`<Container maxWidth="lg" /> `
this Grid instead
Also as I remenber you can add `component={Grid}` prop